### PR TITLE
Lakhveer/teams core

### PR DESCRIPTION
**Pull request by lakhveerk**
1. Adding isSupported function in teams Core test app
2. Moving RegisterFocusEnterHandler  from teamscore to pagesAPI

### CODE REVIEW

**Review by aeunms — APPROVED** [review]
LGTM!

File: apps/teams-test-app/src/components/TeamsCoreAPIs.tsx [hunk 1]
```diff
@@ -82,6 +88,7 @@ const TeamsCoreAPIs = (): ReactElement => (
     <RegisterOnLoadHandler />
     <RegisterBeforeUnloadHandler />
     <RegisterFocusEnterHandler />
```
Comment (MarcinJuraszek) [inline apps/teams-test-app/src/components/TeamsCoreAPIs.tsx:90]: This should move to `pages`, where we already have `returnFocus`.

**Review by MarcinJuraszek — CHANGES_REQUESTED** [review]

File: apps/teams-test-app/src/components/PagesAPIs.tsx [hunk 2]
```diff
@@ -121,6 +122,28 @@ const ReturnFocus = (): React.ReactElement =>
     },
   });
 
+const RegisterFocusEnterHandler = (): React.ReactElement =>
+  ApiWithoutInput({
+    name: 'registerFocusEnterHandler',
+    title: 'Register On Focus Enter Handler',
+    onClick: {
+      withPromise: async setResult => {
```
Comment (MarcinJuraszek) [inline apps/teams-test-app/src/components/PagesAPIs.tsx:130]: These look exactly the same. you don't really need both.

**Review by MarcinJuraszek — CHANGES_REQUESTED** [review]

File: apps/teams-test-app/src/components/PagesAPIs.tsx (hunk 2 above)
Comment (lakhveerk) [reply apps/teams-test-app/src/components/PagesAPIs.tsx:130]: Sure, I can keep only one. btw, I copied it from teamsCoreAPI, where it was added by PR#854 (https://github.com/OfficeDev/microsoft-teams-library-js/pull/854/files)

**Review by MarcinJuraszek — APPROVED** [review]

**Pull request merged**